lib/srv/alpnproxy/local_proxy.go

```diff
@@ -42,6 +52,27 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 )
 
+var (
+	// kubeScheme is the runtime Scheme that holds information about supported
+	// message types.
+	kubeScheme = runtime.NewScheme()
+	// kubeCodecs creates a serializer/deserizalier for the different codecs
+	// supported by the Kubernetes API.
+	kubeCodecs = serializer.NewCodecFactory(kubeScheme)
+)
+
+// Register all groups in the schema's registry.
+// It manually registers support for `metav1.Table` because go-client does not
+// support it but `kubectl` calls require support for it.
+func init() {
```

Comment:
you don't need all of them and you can just initialize for the error function

Reply:
Reduced to just required.